changelog/changes/2024-03-13-systemd-upgrade-from-252-to-255.md

```diff
@@ -0,0 +1 @@
+- Upgraded systemd from version 252 to version 255 ([flatcar/scripts#1679](https://github.com/flatcar/scripts/pull/1679)).
```

Comment:
That entry would rather go into `changelog/updates/` as `- systemd ([255](https://github.com/systemd/systemd/releases/tag/v255))`

Reply:
sure, will do the update. as the two changes are cosmetic, I will wait for the CI to finish before I do the force push.